zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,94 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use zebra_chain::parameters::Network;
+
+/// Whatever
+pub struct Params(pub Network);
+
+impl Params {
+    /// COIN
+    pub const COIN: u32 = 100000000;
+
+    /// SlowStartInterval
+    pub const SLOW_START_INTERVAL: u32 = 20000;
+
+    /// MaxBlockSubsidy
+    pub const MAX_BLOCK_SUBSIDY: u32 = (12.5 * (Self::COIN as f32)) as u32;
+
+    /// PreBlossomPoWTargetSpacing
+    pub const PRE_BLOSSOM_POW_TARGET_SPACING: u32 = 150;
```

Comment:
We might want to use the duration type here:

```suggestion
    pub const PRE_BLOSSOM_POW_TARGET_SPACING: Duration = Duration::from_secs(150);
```

Reply:
We need to divide `PRE_BLOSSOM_POW_TARGET_SPACING` by `POST_BLOSSOM_POW_TARGET_SPACING` right away to get `BLOSSOM_POW_TARGET_SPACING_RATIO` which is actually what we need to be exposed. So i changed  `BLOSSOM_POW_TARGET_SPACING_RATIO` to `Duration` leaving numerator and denominator as `u64` in https://github.com/ZcashFoundation/zebra/pull/1051/commits/fcbbbd4edf43b89bc90ec41bbc091f88a27edc50

Reply:
I think there's a bit of confusion here.

BLOSSOM_POW_TARGET_SPACING_RATIO is just a number. It's not duration (a time in seconds) - it's the unitless ratio between two durations.

We need PRE_BLOSSOM_POW_TARGET_SPACING and POST_BLOSSOM_POW_TARGET_SPACING to be durations, because we will use them in the work module.

Reply:
Ok, got it, sorry about the confusion. I think it is fixed at https://github.com/ZcashFoundation/zebra/pull/1051/commits/f009f702d894b55ff27cf00c7c4370c194a5410e